pkg/resource/cluster.go

```diff
@@ -295,6 +298,47 @@ func (cluster Cluster) IsFresh(fetcher Fetcher) (bool, error) {
 	return cluster.cr.ResourceVersion == actual.ResourceVersion, nil
 }
 
+func (cluster Cluster) LoggingConfiguration() (string, error) {
+	var logYaml string
+	if cluster.Spec().LogConfig != nil {
+		logYamlBytes, err := json.Marshal(cluster.Spec().LogConfig.LogFile)
+		if err != nil {
+			return "", err
+		}
+		var out bytes.Buffer
+		err = json.Indent(&out, logYamlBytes, "", " ")
+		if err != nil {
+			return "", err
+		}
+		logYaml = "\"" + out.String() + "\""
```

Comment:
Have you tested how this behaves if the YAML configuration contains quoted strings?

Reply:
Yes I have verified the changes works with both json and yaml